x/auth/client/cli/query.go

```diff
@@ -198,8 +198,7 @@ $ %s query txs --%s 'message.sender=cosmos1...&message.action=withdraw_delegator
 		},
 	}
 
-	cmd.Flags().StringP(flags.FlagNode, "n", "tcp://localhost:26657", "Node to connect to")
-	cmd.Flags().String(flags.FlagKeyringBackend, flags.DefaultKeyringBackend, "Select keyring's backend (os|file|kwallet|pass|test)")
+	flags.AddQueryFlagsToCmd(cmd)
 	cmd.Flags().Int(flags.FlagPage, rest.DefaultPage, "Query a specific page of paginated results")
 	cmd.Flags().Int(flags.FlagLimit, rest.DefaultLimit, "Query number of transactions results per page returned")
```

Comment:
Can we replace these 2 lines with AddPaginationFlagsToCmd too?

Reply:
Not quite reasonable, AddPaginationFlagsToCmd is used for application state query instead of tendermint txIndex search (maybe we can enhance it), which contains `FlagPageKey` and `FlagOffset`.